README.md

```diff
@@ -1,2 +1,82 @@
-# Metrics-webapp
-This is a React capstone project in which I built a mobile web application to check a list of metrics I used Redux  for state management.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Covid-19 Metrics
+
+![screenshot](./src/assests/mobilescreen.png)
+
+> Covid-19 Metrics Webapp is a web(mobile-focused) application for monitoring covid-19 recorded cases worldwide. It uses data from the Narrativa API.
+
+## Presentation Video
+Here is a video explaining what I have built [Here](https://www.loom.com/share/bec692eae1e04afebfbd702ea8c3bf0b)
+
+## Live demo
+[Covid-19 Metrics-App](https://covid-19-metrics-muwo.herokuapp.com/)
+
+## Built With
+
+- Frameworks / Libraries / Languages
+  ```bash
+  - React 
+  - Redux
+  - CSS
+  - HTML
+  - React Testing Library
+  - Jest
+  - Git
+  - Linters
+  ```
+
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Install
+  -  [Git](https://git-scm.com/downloads)
+  -  [Node](https://nodejs.org/en/download/)
+
+### Usage
+#### Clone this repository
+
+```bash
+$ git clone https://github.com/butlermuwo/Metrics-webapp.git
+$ cd Metrics-webapp
+```
+#### Run project
+
+```bash
+$ npm install
+$ npm start
+```
+
+👤 **Author: Butler Shimaluwani**
+
+[<code><img height="26" style="padding-right:2%;" src="https://cdn.iconscout.com/icon/free/png-256/github-153-675523.png"></code>](https://github.com/butlermuwo)
+[<code><img height="26" style="padding-right:2%;" src="https://upload.wikimedia.org/wikipedia/sco/thumb/9/9f/Twitter_bird_logo_2012.svg/1200px-Twitter_bird_logo_2012.svg.png"></code>](https://twitter.com/ButlerMuwo)
+[<code><img height="26" style="padding-right:2%;" src="https://upload.wikimedia.org/wikipedia/commons/thumb/c/c9/Linkedin.svg/1200px-Linkedin.svg.png"></code>](https://www.linkedin.com/in/butlermuwo/)
+ <a href="mailto:butlermuwo@gmail.com?subject=Hey Butler!"><img height="26" style="padding-right:2%;" src="https://cdn.worldvectorlogo.com/logos/official-gmail-icon-2020-.svg"></a>
+
+<!-- - GitHub: [@ButlerMuwo](https://github.com/butlermuwo)
+- Twitter: [@ButlerMuwo](https://twitter.com/ButlerMuwo)
+- LinkedIn: [@ButlerMuwo](https://www.linkedin.com/in/butlermuwo/) -->
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Acknowledgement
+Original design idea by [Nelson Sakwa on Behance..](https://www.behance.net/gallery/31579789/Ballhead-App-%28Free-PSDs%29) Under the [Creative Commons License](https://creativecommons.org/licenses/by-nc/4.0/)
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Thanks to everyone that helped me, in one way or the other in building this awesome application.
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] [OPTIONAL] I think you forgot to put MIT Licence to your project! Please include it in your project as well! Other than that your project running perfectly!

Reply:
Alright thanks, @Rustamxon7 I'll do that!